pkg/sql/region_util.go

```diff
@@ -2441,6 +2441,7 @@ func (p *planner) optimizeSystemDatabase(ctx context.Context) error {
 		"namespace",
 		"table_statistics",
 		"web_sessions",
+		"region_liveness",
```

Comment:
What do you think of transmuting the crdb_region column into the proper enum type? The loop that processes the regional by row tables contains an example of how to change the type.

We would need to use the same logic in the migration to adjust the column type when creating the table.